common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/CustomerInfoResponseHandler.kt

```diff
@@ -0,0 +1,104 @@
+package com.revenuecat.purchases.common.offlineentitlements
+
+import com.google.gson.internal.bind.util.ISO8601Utils
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.ProductType
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.VerificationResult
+import com.revenuecat.purchases.common.AppConfig
+import com.revenuecat.purchases.common.CustomerInfoFactory
+import com.revenuecat.purchases.common.DateProvider
+import com.revenuecat.purchases.common.DefaultDateProvider
+import com.revenuecat.purchases.common.errorLog
+import com.revenuecat.purchases.strings.CustomerInfoStrings.COMPUTING_OFFLINE_CUSTOMER_INFO_FAILED
+import com.revenuecat.purchases.utils.Iso8601Utils
+import org.json.JSONObject
+import java.util.Date
+
+class CustomerInfoResponseHandler(
```

Comment:
nitpick: it seems like the sole public function of this class is `computeOfflineCustomerInfo`, I'd maybe name it after that instead? Like, it doesn't really _handle_ a "CustomerInfoResponse", instead it just computes a customer info based on the productsFetcher. 
How about `OfflineCustomerInfoCalculator` or something?